react/react1/week1/todo-app/src/ItemList.js

```diff
@@ -0,0 +1,11 @@
+//import "./ItemList.css";
+
+export function ItemList(props) {
```

Comment:
You might check Object destructuring here, and use it like { description, deadline } and thus you might use it directly. A cool feature.